config/routes.rb

```diff
@@ -2,8 +2,15 @@
 
   root "users#index"
   
-  get "users" => "users#index"
-  get "users/:id" => "users#show", as: "user"
-  get "users/:id/posts" => "posts#index", as: "user_post"
-  get "users/:id/posts/:id" => "posts#show", as: "post_detail"
+  resources :users do 
+    resources :posts do
+    end
+  end 
+
+  resources :posts do
+    resources :comments 
+    resources :likes 
```

Comment:
- [ ] Please let's endeavour to only set up routes which we will actually be utilizing. As you can see in the attached image below, doing `resources: likes` sets up a bunch of likes related routes which we have no use of. For the purpose of this milestone, we only need to be able to POST a like. Please only set up a `POST /posts/:post_id/likes` so users can like a post 👍 
![image](https://user-images.githubusercontent.com/46839548/201199624-4348af4c-c483-47f3-a154-d72cd5edafb9.png)